editor/src/clj/editor/console.clj

```diff
@@ -672,6 +674,7 @@
     (view/make-color-scheme
       [["console.reload.successful" (Color/valueOf "#33CC33")]
        ["console.error" (Color/valueOf "#FF6161")]
+       ["console.fatal" (Color/valueOf "#8250df")]
```

Comment:
We didn't take into account Fatal errors, which make it unnoticeable for our users.
I added its own color: 
![CleanShot 2024-10-23 at 17 15 43@2x](https://github.com/user-attachments/assets/140d5562-8248-43fe-87f4-55f82ce25c42)

@vlaaad this part is for your review

Reply:
TBH, I'd prefer if we used the same color for errors. Throughout the editor, we use three colors for severity: blue for info, orange/yellow for warnings, and red for errors. Let us stay consistent.

Reply:
Fixed